src/fides/api/ctl/migrations/versions/ed46521679fb_add_generic_email_connector_types.py

```diff
@@ -0,0 +1,53 @@
+"""add generic email connector types
+
+Revision ID: ed46521679fb
+Revises: 8a71872089e4
+Create Date: 2023-05-19 23:25:48.711050
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "ed46521679fb"
+down_revision = "8a71872089e4"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Add 'generic_consent_email' and 'generic_erasure_email' to ConnectionType enum
+    op.execute("alter type connectiontype rename to connectiontype_old")
+    op.execute(
+        "create type connectiontype as enum('mongodb', 'mysql', 'https', 'snowflake', "
+        "'redshift', 'mssql', 'mariadb', 'bigquery', 'saas', 'manual', 'manual_webhook', "
```

Comment:
whoops, missed this -- but based on the CI test failures, looks like we're missing at least `postgres` here from the new enum values...